packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -120,11 +129,14 @@ export class Autosuggest {
 
   debouncedEmitValue = debounce((value: string) => this.changeEmitter.emit(value), 200);
 
+  inputValue: string = '';
```

Comment:
Ik denk dat dit ook een `@State()` decorator nodig heeft. Het is nu afhankelijk van andere `@Prop()` of  `@State()` properties om een render aan te trappen.